libflux/src/ast/check/mod.rs

```diff
@@ -0,0 +1,77 @@
+use crate::ast::{walk, PropertyKey};
+use crate::ast::SourceLocation;
+use std::fmt;
+
+#[cfg(test)]
+mod tests;
+
+// check will inspect a single node and annotate it with any AST errors.
+// Unlike the ast.Check() in our Go parser, the check function in Rust
+// returns a list of errors generated during the check process PLUS
+// any errors existed before ast.check() is performed.
```

Comment:
nit, Can you update this comment to describe what it does without comparing it to the Go check function? We want this comment to serve as the documentation for the function and its likely that who ever is reading it isn't familiar with the Go behavior.

Reply:
Done